db/osqlcomm.c

```diff
@@ -6436,6 +6436,7 @@ int osql_process_packet(struct ireq *iq, unsigned long long rqid, uuid_t uuid,
             return conv_rc_sql2blkop(iq, step, -1, ERR_NO_SUCH_TABLE, err,
                                      tablename, 0);
         }
+        iq->origdb = iq->usedb;
```

Comment:
Can we instead set `iq->usedbtablevers = iq->usedb->tableversion` in constraints.c? Using `origdb` is a bit confusing to me.